packages/calcite-components/src/components/notice/notice.scss

```diff
@@ -3,7 +3,13 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-notice-width: The width of the component.
+ * @prop --calcite-notice-background: Specifies the background color of the component.
+ * @prop --calcite-notice-close-background-focus: Specifies the background color of the component when focused.
+ * @prop --calcite-notice-close-background-press: Specifies the background color of the component when active.
+ * @prop --calcite-notice-close-text-color-hover: Specifies the background color of component's close button when hovered.
+ * @prop --calcite-notice-close-text-color: Specifies the text color of component's close button.
+ * @prop --calcite-notice-content-text-color: Specifies the content text color of the component.
+ * @prop --calcite-notice-width: Specifies the width of the component.
```

Comment:
I wonder if we can evaluate this for removal. Not sure why we'd need this, vs. the user being able to set this.

Reply:
Yeah WDYT @alisonailea?